rasa/core/agent.py

```diff
@@ -892,7 +887,7 @@ def load_local_model(
         if os.path.isfile(model_path):
             model_archive = model_path
         else:
-            model_archive = get_latest_model(model_path)
+            model_archive = get_model(model_path)
```

Comment:
this change doesn't work like this, `get_model` unpacks the model, but it is unpacked again in https://github.com/RasaHQ/rasa/pull/4625/files#diff-96e44592018d9ea0a68a83fad0d220a5L902

Reply:
Oops, yes, misread it to mean it's directly using the folder as the model.